content/tutorial/tutorial.md

```diff
@@ -213,22 +218,23 @@ class Square extends React.Component {
 }
 ```
 
-Before:
+पहले:
 
 ![React Devtools](../images/tutorial/tictac-empty.png)
 
-After: You should see a number in each square in the rendered output.
+बादमे: आपको प्रदान किए गए आउटपुट में प्रत्येक वर्ग में एक संख्या दिखनी चाहिए।
 
 ![React Devtools](../images/tutorial/tictac-numbers.png)
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/aWWQOG?editors=0010)**
+**[इस बिंदु पर पूर्ण कोड देखें](https://codepen.io/gaearon/pen/aWWQOG?editors=0010)**
```

Comment:
इस बिंदु पर -> literal translation, change to something better

Reply:
how about `अभी तक का पूरा कोड देखें`?

Reply:
cool